covid19/google_data.py

```diff
@@ -39,6 +39,7 @@ def spreadsheet_download_url(url_or_id, file_format):
     )
 
 
+@retry(tries=3, delay=5)
 @cache_memoize(24 * 3600)
 def get_general_spreadsheet(timeout=5):
```

Comment:
Do jeito que está, fica assim:

```python
get_general_spreadsheet = retry(tries=3, delay=5)(cache_memoize(24 * 3600)(get_general_spreadsheet))
```

Acredito que o correto seria os decoradores ao contrário:

```python
get_general_spreadsheet = cache_memoize(24 * 3600)(retry(tries=3, delay=5)(get_general_spreadsheet))
```

```suggestion
@cache_memoize(24 * 3600)
@retry(tries=3, delay=5)
def get_general_spreadsheet(timeout=5):
```

Reply:
Resolvido em ca1f37fa9b0403fdc524a504656b908a5e950a07